Add note on Hardhat (#1324)
MetaMask/metamask-docs: wallet/how-to/manage-networks/add-network.md

# Add a network

:::note
See the [Connect to Hardhat Network](../how-to/run-devnet.md) documentation to run a development network using Hardhat and connect MetaMask to it.
:::

> 

Reviewer: I would make the note less prominent - maybe move to after the screenshots. And update it to something like the following for simplicity:

```
:::note
You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md).
:::
```

Reviewer 2: Can the messaging be a bit more general? 

```
Looking for information on how to add a local development network to MetaMask, such as Hardhat? [Learn how to connect a development network.](...)
```

    </div>
</div>

:::note

> You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md)

Reviewer: ```suggestion
You can also [add a development network](../how-to/run-devnet.md), such as Hardhat, to MetaMask.
```

To address Christian's comment on generalizing the note.

Author: Hi @alexandratran I proposed a slightly different wording.

> To add a local development network like [Hardhat](https://hardhat.org) to MetaMask, see the [connect a development network](../how-to/run-devnet.md) documentation.

Reviewer: ```suggestion
To add a local development network such as [Hardhat](https://hardhat.org) to MetaMask, see [Run a development network](../how-to/run-devnet.md).
```
